activedirectory.sgnl.yaml

```diff
@@ -0,0 +1,229 @@
+# All auth mechanisms specified below must be supported by the specified Adapter.
+auth:
+  - basic:
+      username: "{{Input Required: Username}}"
+      password: "{{Input Required: Password}}"
+entities:
+  User:
+    displayName: User
+    externalId: User
+    description: User Entity in Active Directory
+    pageSize: 999
+    pagesOrderedById: false
+    attributes:
+      - name: dn
+        externalId: dn
+        description: distinguished Name of the user
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: obejctClass
+        externalId: obejctClass
+        type: String
+      - name: sn
+        externalId: sn
+        type: String
+      - name: givenName
+        externalId: givenName
+        type: String
+      - name: distinguishedName
+        externalId: distinguishedName
+        type: String
+      - name: instanceType
+        externalId: instanceType
+        type: Int64
+      - name: whenCreated
+        externalId: whenCreated
+        type: DateTime
+      - name: whenChanged
+        externalId: whenChanged
+        type: DateTime
+      - name: uSNCreated
+        externalId: uSNCreated
+        type: Int64
+      - name: uSNChanged
+        externalId: uSNChanged
+        type: Int64
+      - name: department
+        externalId: department
+        type: String
+      - name: name
+        externalId: name
+        type: String
+      - name: userAccountControl
+        externalId: userAccountControl
+        type: Int64
+      - name: badPwdCount
+        externalId: badPwdCount
+        type: Int64
+      - name: codePage
+        externalId: codePage
+        type: Int64
+      - name: countryCode
+        externalId: countryCode
+        type: Int64
+      - name: badPasswordTime
+        externalId: badPasswordTime
+        type: Int64
+      - name: lastLogoff
+        externalId: lastLogoff
+        type: Int64
+      - name: lastLogon
+        externalId: lastLogon
+        type: Int64
+      - name: pwdLastSet
+        externalId: pwdLastSet
+        type: Int64
+      - name: primaryGroupID
+        externalId: primaryGroupID
+        type: Int64
+      - name: accountExpires
+        externalId: accountExpires
+        type: Int64
+      - name: logonCount
+        externalId: logonCount
+        type: Int64
+      - name: sAMAccountName
+        externalId: sAMAccountName
+        type: String
+      - name: sAMAccountType
+        externalId: sAMAccountType
+        type: Int64
+      - name: userPrincipalName
+        externalId: userPrincipalName
+        type: String
+      - name: objectCategory
+        externalId: objectCategory
+        type: String
+      - name: dSCorePropagationData
+        externalId: dSCorePropagationData
+        type: DateTime
+  Group:
+    description: "Group Entity in Active Directory"
```

Comment:
Please follow the format of "An entity representing a XYZ in an Active Directory" for all entities

Please refer to latest catalogs if needed.